graal-js/src/com.oracle.truffle.js/src/com/oracle/truffle/js/builtins/ConstructorBuiltins.java

```diff
@@ -322,6 +322,7 @@ protected ConstructorBuiltins() {
         // --- not new.target-capable below ---
         TypedArray(0),
         Symbol(0),
+        ModuleBlock(1),
```

Comment:
(minor) ModuleBlock cannot be instantiated, but should have 1 argument? `0` seems more reasonable.

Reply:
The 1 argument came basically from the documentation, as it makes more sense at this point to have 0 arguments, I changed the code.